### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by oherrala** [rust-lang/socket2#265]
There's a fix for OpenBSD in cddb67b8534a46a372beb9fd63c5ce8d1473cbda, but this has not been released. Without this some crates depending on `socket2` are not buildable in OpenBSD.

```
error[E0425]: cannot find value `KEEPALIVE_TIME` in this scope
   --> /home/oherrala/.cargo/registry/src/github.com-1ecc6299db9ec823/socket2-0.4.1/src/sys/unix.rs:871:46
    |
871 |         getsockopt::<c_int>(fd, IPPROTO_TCP, KEEPALIVE_TIME)
    |                                              ^^^^^^^^^^^^^^ not found in this scope

For more information about this error, try `rustc --explain E0425`.
```

**Comment by Thomasdezeeuw** [comment]
Just released v0.4.2.

**Comment by oherrala** [comment]
@Thomasdezeeuw Thanks!
